src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RespondToClaimTest.java

```diff
@@ -73,9 +77,7 @@ public void shouldReturnUnprocessableEntityWhenInvalidResponseIsSubmitted() {
         User defendant = idamTestService.createCitizen();
 
         commonOperations.linkDefendant(
```

Comment:
That call is not going to have any effect because created defendant does not have letter holder ID.

I think there is no point trying to link in that scenario, the processing will fail before link is even assessed.